CMakeLists.txt

```diff
@@ -58,6 +58,15 @@ message(STATUS "RMM: RMM_LOGGING_LEVEL = '${RMM_LOGGING_LEVEL}'")
 # cudart can be linked statically or dynamically
 option(CUDA_STATIC_RUNTIME "Statically link the CUDA runtime" OFF)
 
+option(LOGGING_COMPATIBILITY "Enable backwards compatibility with older logging macros" ON)
```

Comment:
We should prefix these options with `RMM_`

Reply:
Done.